targets/Program.cs

```diff
@@ -11,149 +11,150 @@
 using static Bullseye.Targets;
 using static SimpleExec.Command;
 
-namespace targets
+namespace targets;
+
+internal sealed class Options
 {
-    class Options
-    {
-        [Value(0, MetaName = "target", HelpText = "build target to run; see available with: '--list", Default = "rebuild")]
-        public string Target { get; set; }
+    [Value(0, MetaName = "target", HelpText = "build target to run; see available with: '--list", Default = "rebuild")]
```

Comment:
Why do we need to do these changes? We are not using it

Reply:
It's a simple change to make them compliant with the new enabled rules. Instead of disabling the rules, keeping them enabled.
but I don't need to change the file scoped naming and formatting. I'll undo that so it's clearer the actual change.